torch/csrc/jit/python/python_sugared_value.cpp

```diff
@@ -226,14 +226,15 @@ std::shared_ptr<SugaredValue> CUDAPythonModuleValue::attr(
   const std::unordered_set<std::string> cuda_ops = {
       "current_stream",
       "default_stream",
-      "_current_device",
-      "_set_device",
+      "current_device",
+      "set_device",
       "device_index",
       "device_count",
       "set_stream"};
 
   if (cuda_ops.find(field) != cuda_ops.end()) {
-    return std::make_shared<BuiltinFunction>(Symbol::cuda(field), c10::nullopt);
+    return std::make_shared<BuiltinFunction>(
+        Symbol::_cuda(field), c10::nullopt);
```

Comment:
Instead of renaming the entire namespace, can we just desugar the two problematic ops in a special way? i.e. desugar `cuda.current_device()` to `cuda::_current_device`? You could make your `cuda_ops` set into a map or something.

Still not great, but better than renaming the entire namespace to an unconventional name, in my opinion.

Reply:
Ok, yeah. Changed it now.